vita3k/gui/src/archive_install_dialog.cpp

```diff
@@ -166,7 +170,7 @@ void draw_archive_install_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         ImGui::Separator();
         ImGui::Spacing();
-        ImGui::Checkbox("Delete the archive for the content that failed to install?", &delete_archive_file);
+        ImGui::Checkbox("Delete vpk file of the content that failed to install?", &delete_archive_file);
```

Comment:
remove this, i have changed vpk to archive because is zip and vpk, so is archive file